pom.xml

```diff
@@ -0,0 +1,506 @@
+        <module>storage/spring-cloud-azure-starter-integration-storage-queue/storage-queue-operation</module>
+        <module>storage/spring-cloud-azure-starter-integration-storage-queue/storage-queue-integration</module>
+        <module>storage/spring-cloud-azure-starter-storage-queue/storage-queue-client</module>
+        <module>spring-native/storage-blob-native</module>
+        <!--        Add modules here if you want try Spring Cloud Azure 6.x-->
+<!--        <module>aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server</module>-->
+<!--        <module>aad/spring-cloud-azure-starter-active-directory/aad-web-application-and-resource-server</module>-->
+<!--        <module>aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-web-application</module>-->
+<!--        <module>cosmos/azure-spring-data-cosmos/cosmos-multi-database-multi-account</module>-->
+<!--        <module>cosmos/azure-spring-data-cosmos/cosmos-multi-database-single-account</module>-->
+<!--        <module>aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter</module>-->
+<!--        <module>aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter-stateless</module>-->
+<!--        <module>aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server</module>-->
+<!--        <module>aad/spring-security</module>-->
+      </modules>
+      <properties>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
+        <java.version>17</java.version>
+        <spring.boot.version>3.0.0-RC2</spring.boot.version>
+        <version.spring.cloud>2022.0.0-RC2</version.spring.cloud>
+        <version.spring.cloud.azure>6.0.0-beta.4</version.spring.cloud.azure>
+        <version.thymeleaf-extras-springsecurity6>3.1.1.RELEASE</version.thymeleaf-extras-springsecurity6>
+        <mockito.version>4.8.1</mockito.version>
+      </properties>
+      <dependencyManagement>
+        <dependencies>
+          <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-dependencies</artifactId>
+            <version>${spring.boot.version}</version>
+            <type>pom</type>
+            <scope>import</scope>
+          </dependency>
+          <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-dependencies</artifactId>
+            <version>${version.spring.cloud}</version>
+            <type>pom</type>
+            <scope>import</scope>
+          </dependency>
+          <dependency>
+            <groupId>com.azure.spring</groupId>
+            <artifactId>spring-cloud-azure-dependencies</artifactId>
+            <version>${version.spring.cloud.azure}</version>
+            <type>pom</type>
+            <scope>import</scope>
+          </dependency>
```

Comment:
Is it possible to move these contents out of `profile`? Because these contents exist for both spring-cloud-azure-4 and spring-cloud-azure-6.